config-example.json

```diff
@@ -6,7 +6,8 @@
         },
         "AnotherGitRepo" : {
             "url" : "https://www.github.com/YourOrganization/RepoOne.git",
-            "ms-between-poll": 10000
+            "ms-between-poll": 10000,
+            "excluded-dot-files": true
```

Comment:
Should this be "exclude-dot-files" to match the Go and be a little more intuitive?